jwst/extract_2d/grisms.py

```diff
@@ -447,8 +447,8 @@ def extract_grism_objects(input_model,
                                                  ycenter_model &
                                                  order_model) | tr
                 from gwcs.utils import _toindex
```

Comment:
Not sure why this is here. If not specific reason it can go at the top of the module.

Reply:
Moved to top in latest commit.